runtime/src/lib.rs

```diff
@@ -719,9 +719,11 @@ impl_runtime_apis! {
 			match tx.function {
 				// We want to prevent polluting blocks with a lot of useless invalid data.
 				// TODO perform quick and preliminary data validation
+				#[allow(unused_variables)]
```

Comment:
instead of adding a macro to ignore the unused variables in runtime

Why don't we just do what the warning says and write it like "tags: _, ..." ?

Reply:
you can't in this case, needs to be right names, this fix is clever actually

Reply:
Yep, i used the macro on purpose actually..